--- gmpc-libnotify-0.20.0/src/plugin.c.wiget	2010-03-22 04:30:04.000000000 +0100
+++ gmpc-libnotify-0.20.0/src/plugin.c	2011-04-09 14:56:31.770289205 +0200
@@ -149,15 +149,21 @@ static void libnotify_song_changed(MpdOb
 		if(not == NULL)
 		{
 			//            notify_notification_close(not, NULL);
-			not = notify_notification_new(summary, buffer,NULL, NULL);
+			not = notify_notification_new(summary, buffer,NULL
+#if !defined(NOTIFY_VERSION_MINOR) || (NOTIFY_VERSION_MAJOR == 0 && NOTIFY_VERSION_MINOR < 7)
+					, NULL
+#endif
+					);
 		}
 		else{
 			notify_notification_update(not, summary, buffer, NULL);
 		}
 		notify_notification_set_urgency(not, NOTIFY_URGENCY_LOW);
 
+#if !defined(NOTIFY_VERSION_MINOR) || (NOTIFY_VERSION_MAJOR == 0 && NOTIFY_VERSION_MINOR < 7)
 		if(cfg_get_single_value_as_int_with_default(config, "libnotify-plugin", "attach-to-tray", TRUE))
 			notify_notification_attach_to_status_icon(not, tray_icon2_gsi);
+#endif
 
 		g_free(summary);
 		/* Add the song to the widget */
